unit-tests/func/test-set-option.py

```diff
@@ -86,48 +86,41 @@ def check_color_frame_drops(frame):
 depth_sensor.set_option( rs.option.visual_preset, int(rs.l500_visual_preset.max_range) )
 
 
-
 #############################################################################################
-# Test #2
-
+def test_option_changes( sensor ):
+    options = sensor.get_supported_options()
+    for option in options:
+        try:
+            if sensor.is_option_read_only(option): 
+                continue
+            old_value = sensor.get_option( option )
+            range = sensor.get_option_range( option )
+            new_value = range.min
+            if old_value == new_value:
+                new_value = range.max
+            if not log.d( str(option), old_value, '->', new_value ):
+                test.info( str(option), new_value, persistent = True )
+            set_new_value( sensor, option, new_value )
+            sensor.set_option( option, old_value )
+        except: 
+            test.unexpected_exception()
+            test.fail()
```

Comment:
this line is unnecessary, test.unexpected_exception() will already fail the test

Reply:
Removed